airbyte-integrations/connectors/source-greenhouse/source_greenhouse/components.py

```diff
@@ -109,11 +110,14 @@ def stream_slices(self) -> Iterable[StreamSlice]:
                 sync_mode=SyncMode.full_refresh, cursor_field=None, stream_slice=parent_stream_slice, stream_state=None
             ):
                 parent_state_value = parent_record.get(self.parent_key)
```

Comment:
What if there is no `parent_state_value`?

Reply:
Took another look, and now realizing the variable name is a bit misleading. This is set to the value of the current parent record's primary key, so we can assume it always exists. I've updated the variable name to `parent_primary_key` for clarity.